packages/core/src/Utils/Validator/__tests__/validator.spec.js

```diff
@@ -1,42 +0,0 @@
-import { expect } from 'chai';
```

Comment:
Are we removing all `chai` test files?

Reply:
Just the ones that are not valid anymore (the code is moved out), or the code is so diverged that none of the test cases are true anymore.